test/lib/adapters/FileSystem_write.js

```diff
@@ -45,8 +56,8 @@ test("Write resource", async (t) => {
 	// Write resource content to another readerWriter
 
 	await readerWriters.dest.write(resource);
-	t.notThrows(() => {
-		assert.fileEqual(destFsPath, "./test/fixtures/application.a/webapp/index.html");
+	t.notThrows(async () => {
+		await fileEqual(t, destFsPath, "./test/fixtures/application.a/webapp/index.html");
 	});
```

Comment:
```suggestion
	await t.notThrowsAsync(fileEqual(t, destFsPath, "./test/fixtures/application.a/webapp/index.html"));
```

Should already do the trick

Reply:
doing mass updates without looking at the actual code is not the best idea :-D